src/metabase/driver/sql.clj

```diff
@@ -63,6 +63,17 @@
       (seq referenced-card-ids)
       (update :metabase.models.query.permissions/referenced-card-ids set/union referenced-card-ids))))
 
+(defmulti json-field-length
+  "Return a HoneySQL expression that calculates the length of a JSON field for a given driver.
+  `json-field-identifier` is the Identifier ([[metabase.util.honey-sql-2/Identifier]]) for a JSON field."
+  {:added "0.49.22", :arglists '([driver json-field-identifier])}
+  driver/dispatch-on-initialized-driver
+  :hierarchy #'driver/hierarchy)
+
+(defmethod json-field-length :default
+  [_driver _native-form]
+  ;; we rely on this to tell if the method is implemented for this driver or not
+  :nyi)
```

Comment:
is this check (for `:nyi`) implemented elsewhere?

Reply:
good catch, I removed it in a refactor. Was just thinking I needed to write a test for that... 
I've updated it now in 9c888c131122deca9571dc5280486c3610043dd2

Reply:
I'd make this a var or make this value more recognizable + greppable

Reply:
yeah I could namespace the keyword to be like `:driver.sql/nyi`